sdk/core/core-http/test/defaultHttpClientTests.node.ts

```diff
@@ -12,6 +12,7 @@ import { HttpMockFacade, getHttpMock } from "./mockHttp";
 import { PassThrough, Readable } from "stream";
 import { ReadStream, createReadStream } from "fs";
 import { TransferProgressEvent, WebResource } from "../src/webResource";
+import { fakeTestPassword, fakeTestSecretPlaceholder } from "@azure/test-utils";
```

Comment:
Can we use `fakeTestPasswordPlaceholder` and `fakeTestSecretPlaceholder`? I think in this way, you can try to remove the suppression on these.